### PR TITLE
Fix workload statuses

### PR DESCRIPTION
**Pull request by maciaszczykm**
Fixes https://github.com/kubernetes/dashboard/issues/4261.

Adds https://github.com/swimlane/ngx-charts library.

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -104,6 +104,7 @@
     "@angular/platform-browser": "8.0.3",
     "@angular/platform-browser-dynamic": "8.0.3",
     "@angular/router": "8.0.3",
+    "@swimlane/ngx-charts": "^12.1.0",
```
Comment (floreks) [inline package.json]: Remove "^"

File: src/app/frontend/common/components/workloadstatus/style.scss [hunk 2]
```diff
@@ -15,7 +15,9 @@
 @import '../../../variables';
 
 .kd-graph-container {
-  flex: auto;
```
Comment (dudo) [inline src/app/frontend/common/components/workloadstatus/style.scss]: Why drop flex?

Comment (maciaszczykm) [reply src/app/frontend/common/components/workloadstatus/style.scss]: It's not ready yet as you can see in the title. I still have to fix the styling.

**Comment by maciaszczykm** [comment]
Zrzut ekranu 2019-11-28 o 11 27 35

PTAL

**Pull request merged**